var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -91,7 +91,7 @@ class Mesa(MesonPackage):
 
     # TODO: effectively deal with EGL.  The implications of this have not been
     # worked through yet
-    # variant('egl', default=False, description="Enable the EGL frontend.")
+    variant("egl", default=False, description="Enable the EGL frontend.")
```

Comment:
Do you think the `TODO` comment has been addressed adequately? If so, it can be removed.

Reply:
I am not sure what all of the possible implications are, but this did resolve the drm issues that failed downstream when egl was missing.